src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Utilities/StringExtensions.cs

```diff
@@ -294,21 +267,16 @@ public static bool Any(this string subject, params char[] chars)
             return false;
         }
 
-        public static bool All(this string subject, params char[] chars)
+        public static bool All(this string source, params char[] chars)
         {
-            if (string.IsNullOrEmpty(subject))
-            {
-                return true;
-            }
-
-            if (chars == null || chars.Length == 0)
+            if (string.IsNullOrEmpty(source) || chars == null || chars.Length == 0)
```

Comment:
This is a wrong logic

if (chars == null || chars.Length == 0)
            {
                return false;
            }

Reply:
Why?!! the old logic was wrong, when `source` is `null` it should returns `false` not `true`

Reply:
There was no test before, and that is why the test failed when I use the previous logic